### ISSUE TITLE
Undo

### ISSUE THREAD

**Issue by nthiery** [correctexam/corrigeExamFront#340]
Enable undoing the last action with <kbd>Ctrl</kbd>-<kbd>Z</kbd> for all actions.

Main use case: accidental typing or clicking. For that, recursive undo is probably not needed.

**Comment by barais** [comment]
Huge modifications ;). 

I will discuss with @arnobl to see how to manage it correctly

**Comment by nthiery** [comment]
Yeah, I bet. Even small steps (like taking care of some simpler cases) are helpful!

Thanks for investigating.

**Comment by nthiery** [comment]
Thinking more about it while grading: the most useful part would be when hand drawing
over the copies. Typically you make a stroke and it's not super precise; so you want to redo
it. Currently you need to switch to the erase tool, select the stroke, switch back to the 
drawing tool. Ctrl-Z would be much faster.

**Comment by nthiery** [comment]
Just as an additional data point from the current grading session: I confirm that the main use case is by far undoing the latest stroke. Undoing beyond that is nothing but «good to have», some day.
